CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+# Contributing to GrimoireLab Sorting Hat
+
+These are some general guidelines and information related to how we contribute to 
+GrimoireLab. You can read about it from the [CONTRIBUTING.md](https://github.com/chaoss/grimoirelab/blob/master/CONTRIBUTING.md).
+
+## Changelog Entries
+
+Some of your contributions will require a changelog entry which explains 
+the motivation of the change. These changelog entries should be written 
+to the `releases/unreleased/` directory. These entries would be included 
```

Comment:
I'll move the sentence: `These changelog entries should be written to the `releases/unreleased/` directory.` as the first line after the section **Writing changelog entries". I'd add to that sentence a comment about that the entries are written in YAML format and should follow the next schema.